### PR TITLE
Add hover/focus to non-active tabs

### PR DESCRIPTION
**Pull request by howdu**
Default, hover and focus
Screenshot 2023-05-06 at 13 14 52(1)

### CODE REVIEW

**Comment by zepfietje** [comment]
Could you try making the dark hover a little lighter and the focus match active state?

File: packages/forms/resources/views/components/tabs.blade.php [hunk 1]
```diff
@@ -64,7 +64,7 @@
                 x-bind:tabindex="tab === '{{ $tab->getId() }}' ? 0 : -1"
                 class="filament-forms-tabs-component-button flex items-center gap-2 shrink-0 p-3 text-sm font-medium"
                 x-bind:class="{
-                    'text-gray-500 @if (config('forms.dark_mode')) dark:text-gray-400 @endif': tab !== '{{ $tab->getId() }}',
+                    'text-gray-500 hover:text-gray-800 focus:text-primary-600 @if (config('forms.dark_mode')) dark:text-gray-400 dark:hover:text-gray-200 dark:focus:text-primary-600 @endif': tab !== '{{ $tab->getId() }}',
```
Comment (zepfietje) [inline packages/forms/resources/views/components/tabs.blade.php:67]: Is this separate class for dark mode still needed now?

```suggestion
                    'text-gray-500 hover:text-gray-800 focus:text-primary-600 @if (config('forms.dark_mode')) dark:text-gray-400 dark:hover:text-gray-200 @endif': tab !== '{{ $tab->getId() }}',
```

Comment (howdu) [reply packages/forms/resources/views/components/tabs.blade.php:67]: Yeah I thought that too but found that dark text overwrites the focus
Screenshot 2023-05-07 at 08 55 10

Comment (zepfietje) [reply packages/forms/resources/views/components/tabs.blade.php:67]: Ok, wasn't too sure if this worked, but thanks for testing. 😊

**Review by zepfietje — APPROVED** [review]

**Pull request merged**